### PR TITLE
[BCHDCC-95] Hide footer links and List Your Service button

### PR DESCRIPTION
**Pull request by micaelacunhadev**
## Description

Added 'hidden' class to the 'List Your Services' button and link in the header and to links in the footer to hide them from the page in both the desktop and mobile versions.

## Motivation and Context

[BCHDCC-95](https://app.clickup.com/t/9006094761/BCHDCC-##) - Hide footer links and List Your Service button

      
We want the List Your Services button and links in the footer hidden for now.

## Screenshots

 Desktop view - NOT signed in
Screenshot 2024-11-14 at 3 17 55 PM
Desktop view - signed in
Screenshot 2024-11-14 at 3 19 38 PM

Mobile view - NOT signed in
Screenshot 2024-11-14 at 3 18 22 PM

Mobile view - signed in
Screenshot 2024-11-14 at 3 19 14 PM

## Tests to Run
None

## Internal Checklist

- [ ] My code follows the code style of this project (https://rubystyle.guide/).
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by anna-dorigo — CHANGES_REQUESTED** [review]
Since we are leaving the items there (not deleting only hiding them) for future use, I think would be good to add a comment for future developers to know why there are there but hidden all the time.

**Comment by micaelacunhadev** [comment]
> Since we are leaving the items there (not deleting only hiding them) for future use, I think would be good to add a comment for future developers to know why there are there but hidden all the time.

@anna-dorigo good point! I just added comments to where I added the `hidden` class.

File: app/views/shared/_footer.html.haml [hunk 1]
```diff
@@ -5,7 +5,9 @@
         %img{ src: asset_path(SETTINGS[:site_logo]), alt: "#{t('titles.brand')} Footer Logo" }
       %span.subtitle= SETTINGS[:site_subtitle]
 
-    .footer-links{ role: 'navigation', aria: { label: 'Footer Navigation' } }
+    -# hiding footer links for the time being (11/15/2024)
+    -#  please remove this commment block when revisiting this
```
Comment (anna-dorigo) [inline app/views/shared/_footer.html.haml]:
```suggestion
    -# please remove this comment block when revisiting this
```

File: app/views/shared/_header.html.haml [hunk 2]
```diff
@@ -10,7 +10,9 @@
     %span
   %nav#nav-menu
     %ul.desktop-only
-      %li
+      -# hiding the 'List Your Services' button for the time being (11/15/2024)
+      -#  please remove this commment block when revisiting this
```
Comment (anna-dorigo) [inline app/views/shared/_header.html.haml]:
```suggestion
      -# please remove this comment block when revisiting this
```

File: app/views/shared/_header.html.haml [hunk 3]
```diff
@@ -47,7 +49,9 @@
           %img{ src: asset_path('language_flags/korean_language_flag.png'), alt: 'Korean' }
         %a.translate-link{ href: '#', data: { lang: 'es' }, aria: { label: 'Translate page to Spanish' } }
           %img{ src: asset_path('language_flags/spanish_language_flag.png'), alt: 'Spanish' }
-      %li
+      -# hiding the 'List Your Services' link for the time being (11/15/2024)
+      -#  please remove this commment block when revisiting this
```
Comment (anna-dorigo) [inline app/views/shared/_header.html.haml]:
```suggestion
      -# please remove this comment block when revisiting this
```

**Review by anna-dorigo — APPROVED** [review]
This looks good! I just fixed a typo and an extra white space.

**Pull request merged**